test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -651,6 +651,8 @@ add_specs suite_builder =
 
             Decimal.new "12" . div (Decimal.new "0") . should_fail_with Arithmetic_Error
 
+            ((Decimal.new "1") / (Decimal.new "3")) . should_fail_with Arithmetic_Error
```

Comment:
Why does it fail? I don't understand.

Reply:
This fails because `Decimal` defaults to infinite precision unless you use an explicit `Math_Context` value.

The solution to this might be to use a `Math_Context` by default, but doing so might be unexpected, and lose some of the value of `Decimal`, which is that it attempts to handle precision automatically. I think we'll have to use this in anger before we can make that decision, to decide what the proper defaults should be.

Reply:
I see. I don't think it's a problem for it to fail by default, I just asked because I did not understand.

What is the error message in that case? I think we should try to ensure the error message is clear and ideally proposes the fix (use Math_Context to limit precision). Do you think that would be possible?

Reply:
Done.